root/event/EventMerge.js

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
This file shouldn't technically be added in the areas commit (it's a copy of the areas one, but then later fixed in the events commit, which was a bit confusing).